### ISSUE TITLE
Error:Failed to resolve: com.afollestad:material-dialogs:0.7.6.0 

### ISSUE THREAD

**Issue by PadaliaRushabh** [libreliodev/android#319]
Getting Unsolved dependencies error "Error:Failed to resolve: com.afollestad:material-dialogs:0.7.6.0".

**Comment by sfilatovs** [comment]
I've solved this one by changing the Afollestad version. 
In android-master/appiraterAndroid/build.gradle I've added 

repositories {
    maven { url "https://jitpack.io" }
}

dependencies {
    compile('com.github.afollestad.material-dialogs:core:0.8.5.5@aar') {
        transitive = true
    }

instead of old afollestad command that was there. 

I've also updated all the targetAdkVersion 23 (instead of 21 that was there), compileSdkVersion 23
    buildToolsVersion "23.0.2"
(you need appropriate android sdk for that)

basically, you need to update to the newest versions everywhere where it's possible.

**Comment by avittecoqDAJ** [comment]
Thanks @sfilatovs for this answer

**Comment by zapper112** [comment]
thanks
